docs/startup/hello-samza/versioned/index.md

```diff
@@ -61,7 +61,7 @@ Then, you can continue w/ the following command in hello-samza project:
 {% highlight bash %}
 mvn clean package
 mkdir -p deploy/samza
-tar -xvf ./target/hello-samza-0.13.1-SNAPSHOT-dist.tar.gz -C deploy/samza
+tar -xvf ./target/hello-samza-0.13.1-dist.tar.gz -C deploy/samza
```

Comment:
Yeah this one should point to the version on master (0.13.1-SNAPSHOT). Will revert. See description for changing the versions in other files.